packages/main/src/plugin/webview/webview-panel-impl.ts

```diff
@@ -76,6 +76,10 @@ export class WebviewPanelImpl implements WebviewPanel {
     this.#viewType = panelDetails.viewType;
   }
 
+  get id(): string {
```

Comment:
if id needs exposure then probably to rename `#internalId` to `#id`

and just rename existing
```
get internalId(): string {
```

to 
```
get id(): string {
  this.#id
}
```

Reply:
Good idea